Problems with installing

I am getting errors when installing with npm:

npm install cypress-for-wordpress --save-dev

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /node_modules/cypress-for-wordpress/bin/preperdocs.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/node_modules/cypress-for-wordpress/bin/preperdocs.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
   },
   "files": [
     "src/**",
-    "bin/cywp.js"
+    "bin/**"
   ],
   "directories": {
     "lib": "./src ",

